node-operators/rpc.md

```diff
@@ -0,0 +1,163 @@
+    --db-cache=1024 \
+    --trie-cache-size=1073741824 \
+    --detailed-log-output \
+    --rpc-max-connections=400 \
+    -- \
+    --chain=/chain-network/relay-raw-no-bootnodes-specs.json \
+    --database=paritydb \
+    --rpc-port=9945 \
+    --prometheus-port=9616 \
+    --prometheus-external \
+    --name=relay \
+    --listen-addr=/ip4/0.0.0.0/tcp/30334 \
+    --pool-limit=0 \
+    --db-cache=128 \
+    --state-pruning=2000 \
+    --blocks-pruning=2000 \
+    --rpc-cors=all \
+    --rpc-methods=safe \
+    --unsafe-rpc-external \
+    --sync=fast \
+    --bootnodes=/dns4/frag3-stagenet-relay-val-0.g.moondev.network/tcp/30334/p2p/12D3KooWKvtM52fPRSdAnKBsGmST7VHvpKYeoSYuaAv5JDuAvFCc \
+    --bootnodes=/dns4/frag3-stagenet-relay-val-1.g.moondev.network/tcp/30334/p2p/12D3KooWQYLjopFtjojRBfTKkLFq2Untq9yG7gBjmAE8xcHFKbyq \
+    --bootnodes=/dns4/frag3-stagenet-relay-val-2.g.moondev.network/tcp/30334/p2p/12D3KooWMAtGe8cnVrg3qGmiwNjNaeVrpWaCTj82PGWN7PBx2tth \
+    --bootnodes=/dns4/frag3-stagenet-relay-val-3.g.moondev.network/tcp/30334/p2p/12D3KooWLKAf36uqBBug5W5KJhsSnn9JHFCcw8ykMkhQvW7Eus3U \
+    --bootnodes=/dns4/vira-stagenet-relay-validator-0.a.moondev.network/tcp/30334/p2p/12D3KooWSVTKUkkD4KBBAQ1QjAALeZdM3R2Kc2w5eFtVxbYZEGKd \
+    --bootnodes=/dns4/vira-stagenet-relay-validator-1.a.moondev.network/tcp/30334/p2p/12D3KooWFJoVyvLNpTV97SFqs91HaeoVqfFgRNYtUYJoYVbBweW4 \
+    --bootnodes=/dns4/vira-stagenet-relay-validator-2.a.moondev.network/tcp/30334/p2p/12D3KooWP1FA3dq1iBmEBYdQKAe4JNuzvEcgcebxBYMLKpTNirCR \
+    --bootnodes=/dns4/vira-stagenet-relay-validator-3.a.moondev.network/tcp/30334/p2p/12D3KooWDaTC6H6W1F4NkbaqK3Ema3jzc2BbhE2tyD3YEf84yNLE \
+```
+
+Note that even though the previous command has many parameters, only some of them grouped in the first lines need to be changed to spin up your specific Appchain:
+
+```bash
+    docker run -ti moondancelabs/dancebox-container-chain-evm-templates \
+    /chain-network/container-chain-template-TEMPLATE_TYPE-node \
+    --chain=/chain-network/container-YOUR_APPCHAIN_ID-raw-specs.json \
+    --bootnodes=INSERT_YOUR_APPCHAIN_BOOTNODE \
+    ...
+```
+
+- **TEMPLATE_TYPE** - replace the text with either `frontier` for EVM ContainerChains or `simple` for Substrate ContainerChains
+- **YOUR_APPCHAIN_ID** - replace the text with your Appchain identifier, which is obtained in the [third step](/builders/deploy/dapp/#reserve-appchain-id){target=\_blank} of the registration process
+- **INSERT_YOUR_APPCHAIN_BOOTNODE** - replace the text with the Tanssi provided bootnode, which can be read from the Tanssi Appchain storage on its [Polkadot.js website](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}. The value must be queried from the `dataPreservers`.`bootnodes` storage unit, using your Appchain Id as the option parameter, as shown in the following image:
+
+![Getting the bootnode](/images/node-operators/rpc/rpc-1.webp)
```

Comment:
This image is pretty blurry. Can you try zooming in? I'd also indent it so it's directly under the bullet point that refers to it